packages/tendermint-rpc/src/rpcclients/httpclient.ts

```diff
@@ -65,29 +65,67 @@ export interface HttpEndpoint {
   readonly headers: Record<string, string>;
 }
 
+export interface HttpClientOptions {
+  dispatchInterval: number;
+}
+
+export const defaultHttpClientOptions: HttpClientOptions = { dispatchInterval: 0 };
+
 export class HttpClient implements RpcClient {
   protected readonly url: string;
   protected readonly headers: Record<string, string> | undefined;
+  protected readonly options: HttpClientOptions;
 
-  public constructor(endpoint: string | HttpEndpoint) {
+  private stack: Array<{
```

Comment:
This should be a queue (first in, first out), not a stack (first in, last out). You already implement it that way. But the variable name should be adjusted.